ledger/src/blockstore_processor.rs

```diff
@@ -1614,11 +1608,7 @@ fn confirm_slot_entries(
     let last_entry_hash = entries.last().map(|e| e.hash);
     let verifier = if !skip_verification {
         datapoint_debug!("verify-batch-size", ("size", num_entries as i64, i64));
-        let entry_state = entries.start_verify(
-            &progress.last_entry,
-            replay_tx_thread_pool,
-            recyclers.clone(),
-        );
+        let entry_state = entries.start_verify(&progress.last_entry, replay_tx_thread_pool);
```

Comment:
We should update `poh_verify_elapsed` immediately after this statement as:
1. The result is ready right now
2. If we enter the below `if entry_state.status() == EntryVerificationStatus::Failure` block, we will not update that metric even though we spent time doing it.
    - This was technically a shortcoming of the previous code to handle both CPU and GPU, but we can seemingly fix it now with little hassle

Reply:
see https://github.com/anza-xyz/agave/pull/8967#discussion_r2529588204. finish can return the time